test/to_svg.hpp

```diff
@@ -190,7 +190,7 @@ inline void to_svg(G1 const& g1, G2 const& g2, G3 const& g3,
 
     bg::detail::relate::turns::get_turns
         <
-            G1, G2, point_type, turn_policy
+            G1, G2, turn_policy
```

Comment:
This is the fix for error similar to this one, correct?
```
In file included from libs/geometry/test/algorithms/set_operations/sym_difference/test_sym_difference_linear_linear.hpp:21:0,
                 from libs/geometry/test/algorithms/set_operations/sym_difference/sym_difference_linear_linear.cpp:23:
libs/geometry/test/to_svg.hpp: In function ‘void to_svg(const G1&, const G2&, const G3&, const string&)’:
libs/geometry/test/to_svg.hpp:194:9: error: wrong number of template arguments (4, should be at least 2)
         >::apply(turns, g1, g2, interrupt_policy, strategy);
```

Should `to_svg.hpp` be included if `TEST_WITH_SVG` is not defined?
It is in the following files:
https://github.com/boostorg/geometry/blob/develop/test/algorithms/set_operations/intersection/test_intersection_linear_linear.hpp#L22
https://github.com/boostorg/geometry/blob/develop/test/algorithms/set_operations/difference/test_difference_linear_linear.hpp#L22
https://github.com/boostorg/geometry/blob/develop/test/algorithms/set_operations/sym_difference/test_sym_difference_linear_linear.hpp#L21
https://github.com/boostorg/geometry/blob/develop/test/algorithms/set_operations/union/test_union_linear_linear.hpp#L21

Reply:
That makes sense, I will change it.

Reply:
:heavy_check_mark: